TAGradingServer/account/iframe/file-display.php

```diff
@@ -17,6 +17,12 @@
 	die();
 }
 
+$content_type = getContentType($filename);
+if( substr($content_type, 0, 4) !== "text"){
+    header("Content-type: ".$content_type);
+    header('Content-Disposition: inline; filename="' .  basename($filename) . '"');
+    echo file_get_contents($filename);
```

Comment:
@MasterOdin We talked about this in our meeting today.  I'm worried about security of this new line 24.  What if a student attempts to inject bad stuff with their upload.  We can't wrap an htmlentitiies around this (it breaks pdf viewing).  Your thoughts?